pkg/kubecost/summaryallocation.go

```diff
@@ -0,0 +1,1155 @@
+package kubecost
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/kubecost/cost-model/pkg/log"
+)
+
+// SummaryAllocation summarizes Allocation, keeping only the fields necessary
+// for providing a high-level view of identifying the Allocation (Name) over a
+// defined period of time (Start, End) and inspecting per-resource costs
```

Comment:
I definitely see what you mean. I don't mention `Window` here because, unlike `Allocation`, there is no `Window` on this type at all. You can have a `SummaryAllocation` with no concept of `Window`, feasibly. That's a matter entirely left to the `SummaryAllocationSet` now, which actually feels nicer, but is a departure from the existing precedent.

With that in mind, do you still think it warrants mentioning? I can try to sharpen up the idea a bit, regardless.

Reply:
> That's a matter entirely left to the SummaryAllocationSet now, which actually feels nicer, but is a departure from the existing precedent.

I think it's worth mentioning this in a comment on the type, just because there's a lot of momentum behind the start end vs. window start end concept right now.